istioctl/cmd/auth.go

```diff
@@ -121,6 +124,32 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 			return nil
 		},
 	}
+
+	validatorCmd = &cobra.Command{
+		Use:   "validate <policy-file1,policy-file2,...>",
+		Short: "Validate authentication and authorization policy",
+		Long: `Validate and find incorrectness in authentication and authorization policy files, such as:
```

Comment:
I think we should make the word more neutral, the command checks something that might be wrong, but not necessarily always wrong. Like customer is keeping some ServiceRoleBinding without ServiceRole, but may add the ServiceRole sometime later.

Reply:
Also it seems it only checks authZ for now, so please don't mention authentication in the doc.

Reply:
Will remove the authN part